editor/src/components/inspector/sections/layout-section/layout-section.tsx

```diff
@@ -53,9 +56,16 @@ export const LayoutSection = betterReactMemo('LayoutSection', (props: LayoutSect
   )
 
   return (
-    <>
+    <div
+      onMouseOver={() => {
+        dispatch([setInspectorLayoutSectionHovered(true)], 'everyone')
+      }}
+      onMouseOut={() => {
+        dispatch([setInspectorLayoutSectionHovered(false)], 'everyone')
```

Comment:
Is the hover state still cleared if I mouse over and then `cmd` tab to a different application, move the mouse, then switch back to Utopia?

Reply:
I'll test that and report back.

Reply:
That appears to work fine testing it here.